test/functional/feature_asmap.py

```diff
@@ -26,6 +26,7 @@
 import os
 import shutil
 
+from test_framework.util import assert_equal
```

Comment:
nit: By alphabetical ordering this would go below the following line

Reply:
done